KubeArmor/core/kubeArmor.go

```diff
@@ -311,6 +311,29 @@ func GetOSSigChannel() chan os.Signal {
 	return c
 }
 
+// ======================= //
+// == Container runtime == //
+// ======================= //
+
+// host container runtime available in case of containerized workloads
+func hostContainerRuntime() (string, string) {
```

Comment:
what is the purpose of this function?

Reply:
what is different with kl.GetCRISocket()?

Reply:
Makes sense, I missed that. Thanks